modules.d/90crypt/module-setup.sh

```diff
@@ -143,20 +143,5 @@ install() {
     inst_simple "$moddir/crypt-lib.sh" "/lib/dracut-crypt-lib.sh"
     inst_script "$moddir/crypt-run-generator.sh" "/sbin/crypt-run-generator"
 
```

Comment:
`90crypt/crypt-run-generator.sh` should be moved to systemd also and `parse-crypt.sh` should be split in a cryptsetup and systemd version

Reply:
Hmm does this generator even work or is it necessary? rd.luks.allow-discards <--??? ( where did this come from ) Is it not supposed to be enough that users put rd.luks.options=discard to the kernel command line and the initramfs image doesn't include a etc/crypttab and systemd takes care of rest? Or to rephrase this how much in those scripts are relevant today?